File: Data Structures & Algorithms/is-anagram/submission-0.cpp
#include <vector>
using namespace std;
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int> v1(26,0);
        if(s.length() != t.length()){
            return false;
        }
        for(int i = 0;i<s.length();i++){
            v1[s[i]-97]++;
            v1[t[i]-97]--;
        }
        for(int i = 0;i<v1.size();i++){
            if(v1[i]!=0){
                return false;
            }
        }
        return true;
    }
};

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-0.py
class Solution:
    def findMin(self, nums: List[int]) -> int:
        left, right = 0, len(nums) - 1
        while left < right:
            mid = (left + right) // 2
            if nums[mid] > nums[right]:
                left = mid + 1        # min is strictly right of mid
            else:
                right = mid           # min is at mid or left of it
        return nums[left] 